src/PlacePicker/PlacePicker.js

```diff
@@ -0,0 +1,155 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import type { Place as PlaceType } from './PlacePickerTypes';
+import { StyleSheet } from '../PlatformStyleSheet';
+import { TagsInput } from '../TagsInput';
+import PlacePickerList from './components/PlacePickerList';
+
+type Props = {
+  onChangeText: (value: string) => void,
+  suggestedPlaces: PlaceType[],
+  onPressAdd?: (place: Array<PlaceType>) => void,
+  onPressItem?: (place: Array<PlaceType>) => void,
+  onRemoveSelectedPlace?: () => void,
```

Comment:
Do we even need `onPressAdd `, `onPressItem` and `onRemoveSelectedPlace`?
I'm not 100% sure how will final implementation look like. But what about having just something like `onPlacesChange` which will be called on every add/remove/clear with selected places array in parameter.

Reply:
I will rethink this but I am sure that we will need to have difference callback for `onPressAdd` and `onPressItem`. Because when the user clicks on the `onPressItem` he should be redirected to search with just one selected place.
And if the user clicks to `onPressAdd` it should update Search context and add the Tag to input.

Reply:
anyway I like the idea with `onPlacesChange` callback is more clear how to track changes, so I added callback jus for that. And simplify others.

Reply:
Thanks for the explanation about `pressItem/addItem`. It makes sense to me now.
And thanks for adding `onChange`.